src/private_cuda.h

```diff
@@ -50,6 +59,9 @@ typedef struct _cuda_context {
   gpudata *freeblocks;
   cache *kernel_cache;
   unsigned int enter;
```

Comment:
Move the err member below this one so that they can be packed together and avoid padding.